.github/workflows/release_mac.yml

```diff
@@ -70,7 +70,7 @@ jobs:
 
   test:
     needs: build
-    runs-on: ${{ (matrix.target-architecture == 'x86_64') && 'macos-12' || 'macos-14' }}
+    runs-on: ${{ (matrix.target-architecture == 'x86_64') && 'macos-15' || 'macos-14' }}
```

Comment:
Do we need a different version, or can we merge the two now?

Reply:
Do we need both architectures to create the universal build? Is it possible to use only arm? Maybe relevant https://developer.apple.com/documentation/apple-silicon/building-a-universal-macos-binary

Reply:
I don't know the original considerations, so I can't say at the moment. I also don't have a mac available. I'll check that link

Reply:
I see. This is the test step. I think using macos-latest should suffice? I remember there's some issues with x86 mac when it's macos-14. But I am not certain.